src/main/resources/db/changelog/changes/10-create-shopping-carts-table.yaml

```diff
@@ -0,0 +1,28 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-shopping-carts-table
+      author: anton dudar
+      changes:
+        - createTable:
+            tableName: shopping_carts
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false

```

Comment:
You can delete nullable: false. It's redundant.

Reply:
agree, but it's up to you :)